libtelemetry/src/test/java/com/mapbox/android/telemetry/CertificateBlacklistTest.java

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.android.telemetry;
+
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+
+import org.junit.Test;
+
+import okhttp3.OkHttpClient;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class CertificateBlacklistTest extends MockWebServerTest {
+
+  @Test
+  public void checkDaySinceLastUpdate() throws Exception {
+    Context mockedContext = mock(Context.class);
+    CertificateBlacklist certificateBlacklist = new CertificateBlacklist(mockedContext, "anAccessToken", "AnUserAgent");
```

Comment:
can we decouple ` CertificateBlacklist certificateBlacklist = new CertificateBlacklist(mockedContext, "anAccessToken", "AnUserAgent");` into 
```
@Before
public void setUp() {
....
}
```
that way you don't need to do this with every test method.